src/main/java/com/labsynch/labseer/domain/RDKitStructure.java

```diff
@@ -18,42 +8,7 @@
 @RooJavaBean
 @RooToString
 @RooJson
-@RooJpaActiveRecord(sequenceName = "RDKIT_STRUCTURE_PKSEQ", inheritanceType = "TABLE_PER_CLASS", finders = { "findRDKitStructuresByRegEquals", "findRDKitStructuresByPreRegEquals"})
-public class RDKitStructure {
-
-    @Id
-    @SequenceGenerator(name = "rdkitStructureGen", sequenceName = "RDKIT_STRUCTURE_PKSEQ")
-    @GeneratedValue(strategy = GenerationType.AUTO, generator = "rdkitStructureGen")
-    private Long id;
-
-	public Long getId() {
-		return this.id;
-	}
-
-	public void setId(Long id) {
-		this.id = id;
-	}
-
-    @NotNull
-    @Column(columnDefinition = "CHAR(40)")
-    private String preReg;
-
-    @NotNull
-    @Column(columnDefinition = "CHAR(40)")
-    private String reg;
-
-    @NotNull
-    @Column(columnDefinition = "text")
-    private String mol;
-
-    @NotNull
-    @DateTimeFormat(style="M-")
-    private Date recordedDate;
-
-    public void updateStructureInfo(String mol, String reg, String preReg) {
-        this.setMol(mol);
-        this.setReg(reg);
-        this.setPreReg(preReg);
-    } 
+@RooJpaActiveRecord(sequenceName = "RDKIT_STRUCTURE_PKSEQ", finders = { "findRDKitStructuresByRegEquals", "findRDKitStructuresByPreRegEquals"})
+public class RDKitStructure extends AbstractRDKitStructure {
```

Comment:
Now that you have `AbstractRDKitStructure` and the specific classes seem to extend that, what is the purpose of this `RDKitStructure` class?

Reply:
Ah, I see this is the class name for the parent structure. I think it would be clearer to rename this class `RDKitParentStructure` to make it crystal clear.

Reply:
That makes sense, however, if I make that change now, its going to make for some annoying conflicts as part of my next pull request.  I'd prefer to make this change when I rename everything to be bbchem rather than rdkit.